include/dynamatic/Support/CFG.h

```diff
@@ -97,8 +97,26 @@ bool isBackedge(Value val, Operation *user, BBEndpoints *endpoints = nullptr);
 /// user (the function will assert if that is not the case).
 bool isBackedge(Value val, BBEndpoints *endpoints = nullptr);
 
-/// Determines whether the Handshake operation cannot belong to the implicit CFG
-/// of a Handshake function (i.e., cannot have a basic block attribute).
+/// Data structure to hold all edges belonging to a pair of endpoints. This
+/// constitutes an Arc. srcBB and dstBB can be equal when the arcs are
+/// Backedges. An edge is represented by means of an mlir::OpOperand.
+struct BBArc {
+  unsigned srcBB;
+  unsigned dstBB;
+  mlir::SmallVector<OpOperand *> edges;
+};
+
+/// Define a map from a BB's number to a vector of BBArcs. This can be used, for
```

Comment:
nit: Block comments are generally written with "third-person syntax", so here it would start with "Defines" instead of "Define", as it "This defines...".

This comment applies to all other block comments.